pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -494,6 +494,52 @@ def test_mangled(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    def test_lambda_named_agg(self):
+        # see gh-28467
+        animals = DataFrame(
+            {
+                "kind": ["cat", "dog", "cat", "dog"],
+                "height": [9.1, 6.0, 9.5, 34.0],
+                "weight": [7.9, 7.5, 9.9, 198.0],
+            }
+        )
+
+        result = animals.groupby("kind").agg(
+            mean_height=("height", "mean"),
+            perc90=("height", lambda s: np.percentile(s, q=0.90)),
+        )
+        expected = DataFrame(
+            [[9.3, 9.1036], [20.0, 6.252]],
+            columns=["mean_height", "perc90"],
+            index=Index(["cat", "dog"], name="kind"),
+        )
+
+        tm.assert_frame_equal(result, expected)
+
+    def test_func_named_agg(self):
+        # see gh-28467
+        def myfunc(s):
```

Comment:
Can we just parameterize over the func (lambda and def)?

Reply:
Thought about it.  I think if we make these tests non-OOP, it would make more sense.

Would standout a little amongst all this OOP-ness, but in the name of `pytest` idiom, why not.